lib/game.rb

```diff
@@ -0,0 +1,176 @@
+class Game
```

Comment:
Really nice job refactoring this whole game file. It is very easy to read and follow along. The helper methods are real clean and simple.